website_portal_purchase/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
```

Comment:
I remember that convo regarding the symbol vs the word, which is why I was quoting here. I guess it's a question of whether we upgrade things like this to current OCA style during a migration or not. Current templates have the word as a result of the convo.